index.html

```diff
@@ -27,7 +27,164 @@ <h1 id="name">
+    <img src="img/23.png" alt="">
+    <div class="description-container">
+      <h2 class="project-name">Project name goes here</h2>
+      <div id="language-container">
+        <div class="languages"> <div class="text">HTML/CSS</div></div>
+        <div class="languages"> <div class="text">Ruby on Rails</div></div>
+        <div class="languages"> <div class="text">JavaScript</div> </div>
+      </div>
+      <div class="link-container">
+        <div id="see-this-project">See this project</div>
+        <div id="arrow"><a href="#">&#x2192</a></div>
+      </div>
+    </div>
+  </section>
+
+  <section class="class-container2">
+    <img src="img/24.png" alt="">
+    <div class="description-container">
+      <h2 class="project-name">Project name goes here</h2>
+      <div id="language-container">
+        <div class="languages"> <div class="text">HTML/CSS</div></div>
+        <div class="languages"> <div class="text">Ruby on Rails</div></div>
+        <div class="languages"> <div class="text">JavaScript</div> </div>
+      </div>
+      <div class="link-container">
+        <div id="see-this-project">See this project</div>
+        <div id="arrow"><a href="#">&#x2192</a></div>
+      </div>
+    </div>
+  </section>
+
+  <section class="class-container2">
+    <img src="img/25.png" alt="">
+    <div class="description-container">
+      <h2 class="project-name">Project name goes here</h2>
+      <div id="language-container">
+        <div class="languages"> <div class="text">HTML/CSS</div></div>
+        <div class="languages"> <div class="text">Ruby on Rails</div></div>
+        <div class="languages"> <div class="text">JavaScript</div> </div>
+      </div>
+      <div class="link-container">
+        <div id="see-this-project">See this project</div>
+        <div id="arrow"><a href="#">&#x2192</a></div>
+      </div>
+    </div>
+  </section>
+
+  <section class="class-container2">
+    <img src="img/26.png" alt="">
+    <div class="description-container">
+      <h2 class="project-name">Project name goes here</h2>
+      <div id="language-container">
+        <div class="languages"> <div class="text">HTML/CSS</div></div>
+        <div class="languages"> <div class="text">Ruby on Rails</div></div>
+        <div class="languages"> <div class="text">JavaScript</div> </div>
+      </div>
+      <div class="link-container">
+        <div id="see-this-project">See this project</div>
+        <div id="arrow"><a href="#">&#x2192</a></div>
+      </div>
+    </div>
+  </section>
+
+  <section class="container3">
+    <h1 id="about">
+      About
+    </h1>
+    <p id="myintro">
+      Lorem ipsum dolor sit amet, consectetur adipiscing elit,
+       sed do eiusmod tempor incididunt ut labore et dolore magna aliqua.
+        Ut enim ad minim veniam, 
+        quis nostrud exercitation ullamco laboris nisi ut aliquip ea eiusmod
+    </p>
+    <div class="button">
+      <div class="btn" id="unique">Get my CV</div> 
```

Comment:
**OPTIONAL** - As I mentioned before, please use proper HTML tags, I advise you use a `button` tag here. Please fix this 👍 👍